tests/no-striping.cpp

```diff
@@ -0,0 +1,232 @@
+#include "device/hal/device.manager.h"
```

Comment:
For future context, can you add a comment at the top describing what this test asserts? Maybe add a link to the pr.